plugin/esformatter.vim

```diff
@@ -19,6 +19,17 @@ endif
 
 let loaded_esformatter = 1
 
+function! s:EsformatterCommand()
+  let options = ''
+  if exists('g:esformatter_preset')
+      let options .= '-p ' . shellescape(g:esformatter_preset)
```

Comment:
preset will be deprecated soon (jquery preset will become a plugin in the next few months)